start-oauth-server.sh

```
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+echo "Starting Assisted Service MCP Server with OAuth Authentication..."
+echo
+
+# Load OAuth configuration
+if [ -f oauth-config.env ]; then
+    export $(grep -v '^#' oauth-config.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)